app/src/main/java/com/android/sample/model/activity/ActivitiesRepositoryFirestore.kt

```diff
@@ -34,6 +34,7 @@ open class ActivitiesRepositoryFirestore(private val db: FirebaseFirestore) : Ac
                 .map { document ->
                   Log.d(TAG, "${document.id} => ${document.data}")
                   val data = document.data ?: return@map null // Handle null data gracefully
+                  val images = data["images"] as? List<String> ?: listOf()
```

Comment:
Ok so now the imagees are stored in the firestore repository as strings, and you return an empty list if there is not matches for this field in the database. As I am coding the implementation of loading images form camera, this is good because I did the same concerning profile pictures

Reply:
perfect, for it seems perfect